test/powershell/engine/ParameterBinding/ParameterBinding.Tests.ps1

```diff
@@ -281,6 +281,39 @@
         }
     }
 
+    It "Verify implicit argument conversion uses InvariantCulture" {
+        function Test-Function {
+            param([DateTime] $date)
+            $date
+        }
+
+        function Test-ScriptCmdlet {
+            [CmdletBinding()]
+            param([DateTime] $date)
+            $date
+        }
```

Comment:
Should we add C# cmdlet?

Reply:
We can use `Get-Date -Date` instead, so no need to add a C# cmdlet.
```
PS:162> gcm Get-Date -Syntax
Get-Date [[-Date] <datetime>] ...
```